README.md

```diff
@@ -1,159 +1,199 @@
 🏝 Oasis JsBridge
 ===============
 
-Evaluate JavaScript code and map values, objects and functions between Kotlin/Java and JavaScript on Android.  
+Evaluate JavaScript code and map values, objects and functions between Kotlin/Java and JavaScript on Android.
+
+```kotlin
+val jsBridge = JsBridge()
+val msg: String = jsBridge.evaluate("'Hello world!'.toUpperCase()")
+println(msg)  // HELLO WORLD!
+```
 
 Powered by:
-- [Duktape][duktape] or
-- [QuickJS][quickjs]
+- [Duktape][duktape] (E5 + partial ES2015) or
+- [QuickJS][quickjs] (ES2020)
 
 
 ## Features
 
-Based on [Duktape Android][duktape-android] we added:
- * advanced type inference (generics, reified type parameters)
- * support for non-blocking API, suspending functions (coroutines)
- * higher order functions (lambdas, callback functions)
- * two-way support for JS promises (mapped to Kotlin via Deferred or suspending functions)
- * JsValue class to reference JS objects from Kotlin/Java
- * JsonObjectWrapper for object serialization
- * polyfills for some JS runtime features (e.g. setTimeout, XmlHttpRequest, console)
+ * evaluate JavaScript code from Java/Kotlin
+ * map values, objects and functions between Java/Kotlin and JavaScript
+ * propagate exceptions between JavaScript and Java/Kotlin (including stack trace)
+ * non-blocking API (via coroutines)
+ * support for suspending functions and JavaScript promises
+ * extensions (optional): console, setTimeout/setInterval, XmlHttpRequest, Promise, JS debugger
 
+See [Example](#example-consuming-a-js-api-from-kotlin).
 
-## Usage
-  
-1. [Evaluate JS code](#1-evaluating-js-code)
-1. [Reference any JS value](#2-jsvalue)
-1. [Map JS functions to Kotlin](#3-calling-js-functions-from-kotlin)
-1. [Map Kotlin functions to JS](#4-calling-kotlin-functions-from-js)
-1. [Map JS objects to Java/Kotlin](#5-using-js-objects-from-javakotlin)
-1. [Map Kotlin/Java objects to JS](#6-using-javakotlin-objects-from-js)
 
+## Installation
 
-Minimal hello world:
-```kotlin
-val jsBridge = JsBridge(application.context)
-jsBridge.start()
-jsBridge.evaluateNoRetVal("console.log('Hello world!');")
-jsBridge.release()
+Add jitpack repository (root gradle):
```

Comment:
https://jitpack.io/docs/#badges
We can also add badge on the top of Readme

Reply:
Great idea! Just added the badges for jitpack and github actions.